test/evergreen.yml

```diff
@@ -1015,90 +1026,80 @@ tasks:
       - name: compile
     commands:
       - func: "fetch artifacts"
-      - command: shell.exec
-        params:
-          working_dir: "wiredtiger/build_posix/"
-          script: |
-            set -o errexit
-            set -o verbose
-
-            ${test_env_vars|} $(pwd)/test/cppsuite/run -t base_test -C 'debug_mode=(cursor_copy=true)' -f test/cppsuite/configs/base_test_default.txt -l 2
+      - func: "cppsuite test"
+        vars:
+          test_name: base_test
+          test_config: debug_mode=(cursor_copy=true)
+          test_config_filename: base_test_default.txt
 
   - name: cppsuite-hs-cleanup-default
     tags: ["pull_request"]
     depends_on:
       - name: compile
     commands:
       - func: "fetch artifacts"
-      - command: shell.exec
-        params:
-          working_dir: "wiredtiger/build_posix/"
-          script: |
-            set -o errexit
-            set -o verbose
-
-            ${test_env_vars|} $(pwd)/test/cppsuite/run -t hs_cleanup -C 'debug_mode=(cursor_copy=true)' -f test/cppsuite/configs/hs_cleanup_default.txt -l 2
+      - func: "cppsuite test"
+        vars:
+          test_name: hs_cleanup
+          test_config_filename: hs_cleanup_default.txt
+          test_config: debug_mode=(cursor_copy=true)
 
   - name: cppsuite-search-near-default
     tags: ["pull_request"]
     depends_on:
       - name: compile
     commands:
       - func: "fetch artifacts"
-      - command: shell.exec
-        params:
-          working_dir: "wiredtiger/build_posix/"
-          script: |
-            set -o errexit
-            set -o verbose
-
-            ${test_env_vars|} $(pwd)/test/cppsuite/run -t search_near_01 -f test/cppsuite/configs/search_near_01_default.txt -l 2
-            ${test_env_vars|} $(pwd)/test/cppsuite/run -t search_near_02 -f test/cppsuite/configs/search_near_02_default.txt -l 2
-            ${test_env_vars|} $(pwd)/test/cppsuite/run -t search_near_03 -f test/cppsuite/configs/search_near_03_default.txt -l 2
+      - func: "cppsuite test"
+        vars:
+          test_name: search_near_01
+          test_config_filename: search_near_01_default.txt
```

Comment:
If `test_config` is not set like in this task, will `test/cppsuite/run` handle `-C <blank>` argument input gracefully, e.g. using some default value?

Reply:
Yes, providing `-C` with an empty string or not specifying the `-C` option is the same.